remote-api/src/main/java/org/alfresco/repo/web/scripts/RepositoryContainer.java

```diff
@@ -55,6 +56,7 @@
 import org.alfresco.util.TempFileProvider;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.apache.commons.math3.geometry.spherical.oned.Arc;
```

Comment:
Unwanted import.

Reply:
Good catch - how did that get there (auto-complete on "Arc" ... ) ?

Reply:
Thanks, removing (and I am also guessing that it was auto-complete on "Arc"...)